### PR TITLE
add fixed array serialization/deserialization

### LINKED ISSUE #26: Support fixed size array
Right now only fixed sized byte arrays are supported. According to the spec fixed sized arrays should be supported for any type.

### PR DESCRIPTION
**Pull request by mehtaphysical**
Fixes #26.

This adds serialization for fixed arrays.

### CODE REVIEW

**Comment by mehtaphysical** [comment]
didn't notice there is already #3

File: borsh-ts/test/serialize.test.js [hunk 1]
```diff
@@ -91,9 +91,27 @@ test('serialize/deserialize with class methods', () => {
...
+        ]
+    }]]);
+
+    const buf = borsh.serialize(schema, value);
+    const deserializedValue = borsh.deserialize(schema, Test, buf);
+    
+    expect(buf).toEqual(Buffer.from([5, 0, 0, 0, 104, 101, 108, 108, 111, 5, 0, 0, 0, 119, 111, 114, 108, 100]));
+    expect(deserializedValue.a).toEqual(['hello', 'world']);
+});
+
```
Comment (ailisp) [inline borsh-ts/test/serialize.test.js:111]: This test looks nice. I think we also need a test to test error to serilaize/deserialize when given array has a different array in schema.

**Review by ailisp — APPROVED** [review]
Looks good. Seems CI is broken by fossa. Does your local yarn test work?

**Review by volovyk-s — APPROVED** [review]

**Comment by mehtaphysical** [comment]
added tests for error cases

**Pull request merged**